docs/admin_api/rooms.md

```diff
@@ -483,6 +483,12 @@ The following fields are returned in the JSON response body:
 
 ## Undoing room shutdowns
 
+### If the `block` or `purge` option is `false`
```

Comment:
```suggestion
**If the `block` or `purge` option is `false`**
```

I would prefer to make it bold only rather than as a heading with an entry in the table of contents